cmd/cmd.go

```diff
@@ -1,29 +1,13 @@
 package cmd
 
 import (
-	"path/filepath"
-	"regexp"
-	"strings"
+	"log"
+	"os"
 )
 
-const (
-	msgPleaseAuthenticate = "You must be authenticated. Run `exercism configure --key=YOUR_API_KEY`."
-)
-
-func isTest(path string) bool {
-	ext := filepath.Ext(path)
-	if ext == ".t" {
-		return true
-	}
-
-	file := filepath.Base(path)
-	name := file[:len(file)-len(ext)]
-	if name == "test" || name == "spec" {
-		return true
-	}
-	return regexp.MustCompile(`[\._-]?([tT]est|[sS]pec)`).MatchString(name)
-}
-
-func isREADME(path string) bool {
-	return strings.Contains(path, "README")
+// Bail handles exitable errors in commands.
+// TODO: figure out what goes here.
+func Bail(err error) {
```

Comment:
Is it worth including the `if err != nil` bit in this function as well, maybe make it a `CheckError` function instead.
Then this
```
if err != nil {
    Bail(err)
}
```
could become this
```
CheckError(err)
```